### ISSUE TITLE
VSCodeInsider_DefaultSetting_ScreenReader:Narrator does not announces error message"git  repository URL  must be valid".

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/vscode#88966]
**Environment Details:​**
VSCode Version: 1.42.0-insider (System setup)
OS Version: Win10

**Repro Steps:​**
1.Open VS Code insider 
2. Navigate till "Extension" button and search for the "Visual Studio Online"
3. Click on "Visual Studio Online" and navigate till "Remote Explorer" ,click on it.
4. Navigate till "+" button beside the  "VS ONLINE" control and click on it .
5. Select any existing environment and then again click on the "+" button and navigate till "Default setting" ,click on it.
6.Start the Narrator.
7. text any random text in the search filed.
8. Observe Narrator does not announces error message.
![image](https://user-images.githubusercontent.com/55909580/72719703-567b1580-3b9e-11ea-99ec-fc027499f7c8.png)

**Actual:**
Narrator does not announces error suggestion "git  repository URL  must be valid" after entering invalid text in the "search filed".

**Expected:​**
Narrator should announces error suggestion "git  repository URL  must be valid" after entering invalid text in the "search filed".

**Same issue Observed for the below scenario:**
**1.Customer Setting:**
1.Open VS Code insider 
2.Navigate till "Extension" button and search for the "Visual Studio Online"
3.Click on "Visual Studio Online" and navigate till "Remote Explorer" ,click on it.
4.Navigate till "+" button beside the  "VS ONLINE" control and click on it .
5.Select any existing environment and then again click on the "+" button and navigate till "Custom setting" ,click on it.
![image](https://user-images.githubusercontent.com/55909580/72719655-419e8200-3b9e-11ea-8019-18f00da2ec3c.png)

**​User Impact:​**
Screen reader user does not get updated suggestion information about the control if it does not announces.
[88966_VSCodeInsider_DefaultSetting_ScreenReader.pptx](https://github.com/microsoft/vscode/files/4085710/88966_VSCodeInsider_DefaultSetting_ScreenReader.pptx)

**Comment by isidorn** [comment]
We do not support narrator. Please try NVDA

**Comment by vinodbk197** [comment]
@isidorn 
This issue is not observed with NVDA. 
This issue is specific to Narrator?

**Comment by isidorn** [comment]
Here's more about our accessibilty statment https://code.visualstudio.com/docs/editor/accessibility
We do not test Narrator since it is not open source, thus we are currently not interested in Narattor issues. Thank you
